### ISSUE TITLE
Can't convert a datetime object that have a tzinfo property to pendulum instance

### ISSUE THREAD

**Issue by decko** [sdispater/pendulum#38]
Trying to convert a datetime instance which have a tzinfo property, to a pendulum instance returns an error.

```python
import pendulum
import pytz

from datetime import datetime

date = datetime.now()
date = date.replace(tzinfo=pytz.utc)
pendulum_date = pendulum.instance(date)
```
returns:
```
AttributeError: 'UTC' object has no attribute 'upper'
```

**Comment by sdispater** [comment]
Yes, this is currently a limitation of pendulum. I mention it in #29 :

> Be aware that you can only pass a naive-datetime or a datetime that has been localized by pendulum (see https://pendulum.eustace.io/docs/#using-the-timezone-library-directly) for now.

In your case, you could do:

```python
import pendulum
import pytz

from datetime import datetime

date = datetime.now()
date = date.replace(tzinfo=pytz.utc)
date = pendulum.timezone('UTC').convert(date)
pendulum_date = pendulum.instance(date)
```

Or, better:

```python
import pendulum
import pytz

from datetime import datetime

date = datetime.now()
pendulum_date = pendulum.instance(date)
```

It will give you the same result since pendulum will default to `UTC` for naive-datetime.
Alternatively, you can set the timezone of your naive-datetime:


```python
import pendulum
import pytz

from datetime import datetime

date = datetime.now()
pendulum_date = pendulum.instance(date, tz='Europe/Paris')
```

I will improve the `instance()` method to handle datetime-aware datetimes that have a `tzinfo` not set by pendulum.

**Comment by decko** [comment]
Thanks @sdispater. My problem is Django queryset returning datetime objects with tzinfo set. Thanks anyway.

**Comment by sdispater** [comment]
The improvements to `instance()` should ship with the next major release (0.6).

**Comment by sdispater** [comment]
Fixed in the new `0.6.0` :-)
